src/data.test.ts

```diff
@@ -153,6 +153,17 @@ describe('Dictionary', () => {
     expect(result!.matchLen).toBe(4);
     result = await sharedDict.wordSearch({ input: '行こー' });
     expect(result!.matchLen).toBe(3);
+
+    // A bug was matching input 'クリスチーナ' to entry 'クリスチアナ'
+    result = await sharedDict.wordSearch({
+      input: 'クリスチーナ',
+      doNames: true,
+    });
+    let christiannaRegex = /クリスチアナ/g;
+    let didFindChristiana = result!.data
+        .map((entry) => christiannaRegex.test(entry[0]))
+        .reduce((matchEntryA, matchEntryB) => matchEntryA || matchEntryB);
```

Comment:
I think you could simplify this a little by using [`some()`](https://developer.mozilla.org/docs/Web/JavaScript/Reference/Global_Objects/Array/some), e.g.

```typescript
    const didFindChristiana = result!.data.some((entry) =>
      christiannaRegex.test(entry[0])
    );
```

Reply:
Ah nice! I was looking for something cleaner but didn't find. This will help a lot!
I will also change the `christiana` `christianna` difference in spelling to make it more 統一.

Reply:
Thank you!